specification/cognitiveservices/data-plane/AnomalyDetector/readme.go.md

```diff
@@ -0,0 +1,26 @@
+## Go
+
+These settings apply only when `--go` is specified on the command line.
+
+``` yaml $(go)
+go:
+  license-header: MICROSOFT_APACHE_NO_VERSION
+  namespace: anomalydetector
+  clear-output-folder: true
+```
+
+### Go multi-api
+
+``` yaml $(go) && $(multiapi)
+batch:
+  - tag: release_1_0
+```
+
+### Tag: release_1_0 and go
+
+These settings apply only when `--tag=release_1_0 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'release_1_0' && $(go)
+output-folder: $(go-sdk-folder)/services/cognitiveservices/v1.0/$(namespace)
```

Comment:
Please update this to `$(go-sdk-folder)/services/preview/cognitiveservices/v1.0/$(namespace)` as this is from a preview swagger.

Reply:
What about other language?

Reply:
Sorry I don't know, the other SDK language folks will have to comment.